dashboard/app/controllers/projects_controller.rb

```diff
@@ -291,9 +291,10 @@ def show
     end
 
     iframe_embed = params[:iframe_embed] == true
+    iframe_embed_app_and_code = params[:iframe_embed_app_and_code] == true
     sharing = iframe_embed || params[:share] == true
```

Comment:
I dont think "sharing" is a good name for this variable/option, since we /don't/ want it set for iframe_embed_app_and_code but iframe_embed_app_and_code is a flag that we also set for "sharing" purposes

Reply:
agreed, this is not a great name. it could be helpful to see a quick gsheet breakdown of what each of these settings generally do, even if it is not 100% inclusive.

Reply:
I think starting a spreadsheet to track is a great idea! I think I should start it with the flags I have become familar with over the course of working on this